### PR TITLE
feat(costmodels): migrate to pf4 datatoolbar in the details page

### PR DESCRIPTION
**Pull request by boaz0**
related #1353 

image

image

image

### CODE REVIEW

File: src/pages/costModelsDetails/components/costModelsDetailsToolbar.tsx [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+      ))}
+    </Select>
+  );
+};
+
+type PrimaryFilterBaseProps = InjectedTranslateProps &
+  Omit<SelectFilterProps, 'options'>;
+
+const PrimaryFilterBase: React.SFC<PrimaryFilterBaseProps> = ({
+  t,
```
Comment (dlabrecq) [inline src/pages/costModelsDetails/components/costModelsDetailsToolbar.tsx]: We shouldn't have to provide `t` if the base components each use `translate`.

**Comment by boaz0** [comment]
@dlabrecq can you take a second look? thanks.

File: src/pages/costModelsDetails/components/costModelsDetailsToolbar.tsx [hunk 2]
```diff
@@ -0,0 +1,387 @@
...
+    deleteChip?: () => void;
+    render: () => React.ReactNode;
+  }[];
+  onClearAll: () => void;
+  onRemove: (category: string, chip: string) => void;
+  chips: { [k: string]: string | string[] };
+}
+
+const CostModelsDetailsToolberBase: React.SFC<CostModelsDetailsToolberBaseProps> = ({
+  t,
```
Comment (dlabrecq) [inline src/pages/costModelsDetails/components/costModelsDetailsToolbar.tsx]: Can this be replaced with `translate`?

Comment (boaz0) [reply src/pages/costModelsDetails/components/costModelsDetailsToolbar.tsx]: look at line 385-387:
https://github.com/project-koku/koku-ui/pull/1354/files/51e8b478c1eb6ab32210fd37a490a262205c1a08#diff-0e778d7b2fb39e96a09272e510669ff3R385-R387

**Pull request merged**